pyccel/codegen/printing/ccode.py

```diff
@@ -2272,6 +2275,15 @@ def _print_Program(self, expr):
     def _print_CustomDataType(self, expr):
         return "struct " + expr.name
 
+    def _print_Del(self, expr):
```

Comment:
Can you explain the difference between `Del` and `Deallocate`? For me they are conceptually the same. I think it would make more sense to convert `Del` nodes to `Deallocate` nodes in the semantic stage and not have a `_print_Del` here. This will avoid duplication/confusion

Reply:
Fixed https://github.com/pyccel/pyccel/pull/1508/commits/da631a38dd2c3cc6cc44aa5c9962d3d747ccaa36.